app/middleware/locals.js

```diff
@@ -14,6 +15,10 @@ module.exports = config =>
     res.locals.origin = queryMapper.mapServiceChoice(req.query);
     res.locals.location = req.query.location;
     res.locals.locationHeading = queryMapper.getLocationHeading(req.query);
+    if (res.locals.location) {
```

Comment:
Suggest to push this `if` into `queryMapper.getResultsHeading` in order to simply this module and make it easier to have unit tests against the condition in the mapper.